src/config/userTypes.js

```diff
@@ -0,0 +1,3 @@
+export const USER_TYPE_VIEWER = 'observer';
```

Comment:
Why not use an enum? And is it intended to have `VIEWER` and `obsever`?
```
USER_TYPES = {
  VIEWER: 'observer',
  EDITOR: 'editor',
  OWNER: 'owner'
}
```

Reply:
👍